packages/ui/src/lib/NativeBase/Navbar/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { Image, Box, HamburgerIcon, HStack, Button, Square } from "native-base";
+import PropTypes, { InferProps } from "prop-types";
+import React, { useReducer } from "react";
+import MenuItem, { menuItemPropTypes } from "./menuItem";
+
+const navbarPropTypes = {
+  items: PropTypes.arrayOf(PropTypes.shape(menuItemPropTypes)).isRequired,
+  logo: PropTypes.shape({
+    source: PropTypes.string.isRequired,
+    url: PropTypes.string.isRequired,
+  }).isRequired,
+};
+
+type NavbarProps = InferProps<typeof navbarPropTypes>;
+
+const style = {
+  navbar: {
+    height: ["50px", "50px", "70px"],
+    px: ["15px", "15px", "30px"],
+  },
+  logo: {
+    width: ["101px", "101px", "144px"],
+    height: ["35px", "35px", "50px"],
+  },
+};
+
+const display = {
+  flexOnDesktop: ["none", "none", "flex"],
+  flexOnMobile: ["flex", "flex", "none"],
+};
+
+export default function Navbar({ items, logo }: NavbarProps) {
+  const [isMenuOpen, toggleMenu] = useReducer(
+    (isOpen: boolean) => !isOpen,
```

Comment:
Because [`_.negate`](https://lodash.com/docs/4.17.15#negate) only accept function, it have to be `useReducer(negate(isOpen => isOpen), false)`. I don't think this is better.

Reply:
oh, my bad, I thought there is a negate fn.